kolibri/plugins/facility/assets/src/routes.js

```diff
@@ -124,4 +127,19 @@ export default [
       }
     },
   },
+  {
+    path: '/:facility_id?/:device_id?/managesync',
+    component: ManageSyncSchedule,
+    name: 'ManageSyncSchedule',
+  },
+  {
+    path: '/:facility_id?/:device_id?/editsyncschedule/?',
+    component: editDeviceSyncSchedule,
+    name: 'EditSyncSchedule',
+  },
+  {
+    path: '/:facility_id?/:device_id?/editdevice',
```

Comment:
What is the difference between this and the route above? I think we only need two routes, one for listing all the scheduled syncs, and one for creating/editing the scheduled sync for a specific device?